### ISSUE TITLE
Test GKE guide

### ISSUE THREAD

**Issue by tgraf** [cilium/cilium#6879]
Guide: https://docs.cilium.io/en/latest/gettingstarted/k8s-install-gke/

- [ ] Write down unclear steps in the documentation
- [ ] Report failures

**Comment by nebril** [comment]
Newest k8s version available in GKE is 1.11, we might want to default link to `cilium-with-node-init.yaml` for version 1.11 to be visible instead of 1.13.

I encountered https://github.com/cilium/cilium/issues/6807 , but cluster converged to working state eventually (after about 6 minutes).

We might want to tell our users that it's possible for Cilium pods to restart one or two times before everything starts working and give them a timeframe (up to 10 minutes would be the upper bound I guess).

One issue I found is that `l7-default-backend` pod is stuck in CrashLoopBackOff state. Deleting it solves the issue, the pod probably was not managed by Cilium.
